workers/unity/Packages/io.improbable.gdk.core/Subscriptions/EntityGameObjectLinker.cs

```diff
@@ -139,7 +142,7 @@ public void UnlinkGameObjectFromEntity(EntityId entityId, GameObject gameObject)
             if (linkComponent != null)
             {
                 linkComponent.IsValid = false;
-                linkComponent.EntityId = new EntityId(0);
+                linkComponent.EntityId = new EntityId(id: 0);
```

Comment:
not complaining - but why

Reply:
Oh this is just my linter settings I think